.ci/generate_legacy.jenkinsfile

```diff
@@ -215,7 +228,7 @@ node('Linux') {
 
                 String containerID = get_container_id()
 
-                stage("Legacy - ${compiler.toUpperCase()} ${dir} deploy") {
+                stage("Legacy - ${app.toUpperCase()} ${dir} deploy") {
```

Comment:
Here it was failing: `compiler` was not defined in this scope

Reply:
I love how predictable jenkinsfile are ❤️ 

![](https://i.pinimg.com/564x/94/f0/cb/94f0cb21bd56b93e42b4e712200eced6--perspective-quotes-eren.jpg)